app/templates/main/browse.html

```diff
@@ -0,0 +1,270 @@
+                  <label class="govuk-label govuk-date-input__label filters-form__date__label" for="date-from-month">
+                    (MM)
+                  </label>
+                </div>
+              </div>
+              <div class="govuk-date-input__item">
+                <div class="govuk-form-group">
+                  <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="date-from-year"
+                    name="date-from-year" type="text" inputmode="numeric">
+                  <label class="govuk-label govuk-date-input__label filters-form__date__label" for="date-from-year">
+                    (YYYY)
+                  </label>
+                </div>
+              </div>
+            </div>
+          </fieldset>
+        </div>
+
+        <div class="govuk-form-group date-form filters-form__group filters-form__date-to-group">
+          <p class="govuk-body-m date-form browse__body filters-form__body-date">To date</p>
+          <fieldset class="govuk-fieldset" role="group" aria-describedby="date-to">
+            <div class="govuk-date-input" id="date-to">
+              <div class="govuk-date-input__item">
+                <div class="govuk-form-group">
+                  <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="date-to-day"
+                    name="date-to-day" type="text" inputmode="numeric">
+                  <label class="govuk-label govuk-date-input__label filters-form__date__label" for="date-to-day">
+                    (DD)
+                  </label>
+                </div>
+              </div>
+              <div class="govuk-date-input__item">
+                <div class="govuk-form-group">
+                  <input class="govuk-input govuk-date-input__input govuk-input--width-2" id="date-to-month"
+                    name="date-to-month" type="text" inputmode="numeric">
+                  <label class="govuk-label govuk-date-input__label filters-form__date__label" for="date-to-month">
+                    (MM)
+                  </label>
+                </div>
+              </div>
+              <div class="govuk-date-input__item">
+                <div class="govuk-form-group">
+                  <input class="govuk-input govuk-date-input__input govuk-input--width-4" id="date-to-year"
+                    name="date-to-year" type="text" inputmode="numeric">
+                  <label class="govuk-label govuk-date-input__label filters-form__date__label" for="date-to-year">
+                    (YYYY)
+                  </label>
+                </div>
+              </div>
+            </div>
+          </fieldset>
+        </div>
+
+
+        <div class="filters-form__buttons">
+          <button type="button" class="govuk-button govuk-button__filters-form-apply-button" data-module="govuk-button">
+            Apply filters
+          </button>
+          <p class="govuk-body-m filters-form__clear--button"><a href="#">Clear all filters</a></p>
```

Comment:
Is `<a href="#">` right? Would a `<button type="reset">` work? Just thinking that this is a link that doesn't take you anywhere and would jump you to the top of the page.

Reply:
Changed to a button, this will have further work during the filters development